.github/CONTRIBUTING.md

```diff
@@ -20,16 +20,72 @@ Pull requests are the best way to propose changes to the codebase (we use [Githu
 
 We use GitHub issues to track public bugs. Report a bug by [opening a new issue](https://github.com/BLACC-Smith/website/issues/new/choose); it's that easy!
 
-## Write bug reports with detail, background, and sample code
+## Create a bug report 
 
-**Great Bug Reports** tend to have:
+Select **Create Issue** from the repository
 
-- A quick summary and/or background
-- Steps to reproduce
-  - Be specific!
-  - Give sample code if you can
-- What you expected would happen
-- What actually happens
-- Notes (possibly including why you think this might be happening, or stuff you tried that didn't work)
+ [![Select button for issue](images/new-issue.png)](https://github.com/blaccsmith/website/issues)
 
-People _love_ thorough bug reports. I'm not even kidding.
+ Select **Get Started** for Bug Report.
+
+  [![Select button for bug report](images/bug-and-feature-request.png)](https://github.com/blaccsmith/website/issues)
+
+Complete the following information in the form. Do not create a new file.
+
+```txt
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Where is this happening **
+
+- [ ] Desktop
+- [ ] Mobile
+
+**Additional context**
+Add any other context about the problem here.
+```
+
+## Request a new feature
+
+To request a new feature follow the same path as a bug report and select **Getting Started** for feature request.
+
+[![Select button for new feature](images/bug-and-feature-request.png)](https://github.com/blaccsmith/website/issues)
```

Comment:
Same as above comment

Reply:
Hmm that's just supposed to be a description for the image. I will remove it.